packages/appstore/src/components/total-assets/index.ts

```diff
@@ -0,0 +1,4 @@
+import TotalAssets from './total-assets';
```

Comment:
Can we use `totalBalance` as the name instead of `Asset`?
Is there any reason behind using `asset` as the name?

Reply:
@matin-binary it is the name that is mentioned in design
![Uploading image.png…]()